src/analysisd/state.h

```diff
@@ -130,47 +141,47 @@ void w_inc_received_events();
 /**
  * @brief Increment syscheck decoded events counter
  */
-void w_inc_syscheck_decoded_events();
+void w_inc_syscheck_decoded_events(char * agent_id);
```

Comment:
Add the `agent_id` parameter to the function description. Repeat this in all the functions.